src/pages/TabHome.css

```diff
@@ -22,7 +22,6 @@ ion-button {
 }
 
```

Comment:
This is the hover style. It does not change, so it can be removed.

Reply:
Removed